jetty-ee10/jetty-ee10-maven-plugin/src/it/jetty-start-mojo-it/jetty-simple-webapp/src/config/login.xml

```diff
@@ -0,0 +1,6 @@
+jetty: MD5:164c88b302622e17050af52c89945d44,user
```

Comment:
Why is this file named `login.xml` when its contents are not an XML file?

Reply:
Looks like  a typo to me.